sections/dashboard/SynthBalancesTable/SynthBalancesTable.tsx

```diff
@@ -60,14 +60,14 @@ const SynthBalancesTable: FC<SynthBalancesTableProps> = ({ exchangeTokens }) =>
 	const { t } = useTranslation();
 	const { synthsMap } = Connector.useContainer();
 	const pastRates = useRecoilValue(pastRatesState);
-	const exchangeRates = useAppSelector(selectExchangeRates);
+	const prices = useAppSelector(selectPrices);
 	const { synthBalances } = useAppSelector(selectBalances);
 
 	const synthTokens = useMemo(() => {
 		return synthBalances.map((synthBalance: SynthBalance) => {
 			const { currencyKey, balance, usdBalance } = synthBalance;
 
-			const price = exchangeRates && exchangeRates[currencyKey];
+			const price = prices && prices[currencyKey].offChain;
```

Comment:
Should this be onchain as well? Spot balances would trade via atomic swaps, which don't use offchain oracle price.